orchagent/bufferorch.cpp

```diff
@@ -11,8 +11,6 @@ extern sai_queue_api_t *sai_queue_api;
 extern sai_switch_api_t *sai_switch_api;
 extern sai_buffer_api_t *sai_buffer_api;
 
-extern PortsOrch *gPortsOrch;
-
```

Comment:
Thanks for Ying's reply. I do agree that as a feature development, a merge instead of a squash is a better choice. By the way, for this PR, are there concerns moving the global into the constructor? I think here we could still leave it as it is.